syntax="proto3";

package grpc;

message HelloRequest {
	string name = 1;
}

message HelloReply {
	string message = 1;
}

service HelloSvc {
	rpc sayHello(HelloRequest) returns(HelloReply);
}